pkg/plugins/config/config.go

```diff
@@ -46,12 +46,16 @@ type Cfg struct {
 	Features plugins.FeatureToggles
 
 	AngularSupportEnabled bool
+
+	StackID               string
+	InstallToken          string
+	ExternalManageEnabled bool
```

Comment:
I think it may be possible to get away without adding these to this config - we try to reserve this struct for core internal configuration where possible - so I think we can just inject `*setting.Cfg` where we want this info instead. WDYT?